README.md

```diff
@@ -87,6 +87,43 @@ export default ({ a, b }) => {
 };
 ```
 
+### Exporting multiple worker functions
+
+A single worker file may export multiple named handler functions.
+
+```js
+'use strict';
+
+function add({ a, b }) { return a + b; }
+
+function multiply({ a, b }) { return a * b; }
+
+add.add = add;
+add.multiply = multiply;
```

Comment:
Are these two lines leftover from a previous implementation?

```suggestion
```

Reply:
No, actually the export line is wrong... It should export add as the default...